templates/vue/cypress/integration/lightbox/video.spec.js

```diff
@@ -42,6 +44,10 @@ describe("Video", () => {
       })
       cy.closeLightbox()
 
+      cy.logout().visitTapestry()
+
+      cy.login(roles.SUBSCRIBER).visitTapestry()
```

Comment:
Wondering if there is a particular reason that this test needs to view the video as a subscriber. I tried commenting out line 49 and re-running this particular test, and it is still passing.

Also, if we are to keep line 49, we should probably remove the call to `visitTapestry()` after `logout()` so we avoid visiting twice.

Reply:
I think viewing as a visitor works as well. I will make a commit for that.